tasks/generate_templates.yml

```diff
@@ -78,10 +35,10 @@
     dest: "{{ kops_default_build_directory }}/{{ cluster.name }}/kops-update.sh"
     mode: 0755
 
-- name: generate kops-info.sh
+- name: generate kops-validate.sh
```

Comment:
the tasks generating the bash scripts (create, info, delete & validate) can be condensed with a `loop`.